.circleci/test-deploy.yml

```diff
@@ -57,12 +81,32 @@ workflows:
           working-directory: examples/angular-app
```

Comment:
Should we include something noting this is CT? In the name maybe?

Reply:
Fixed as part of c17814aa103a4b7c21984f679a612553e746cc07